src/dynarec/la64/dynarec_la64_emit_math.c

```diff
@@ -1038,17 +1038,17 @@ void emit_adc32(dynarec_la64_t* dyn, int ninst, rex_t rex, int s1, int s2, int s
 
     if (la64_lbt) {
         if (rex.w)
-            ADC_D(s1, s1, s2);
+            ADC_D(s3, s1, s2);
         else
-            ADC_W(s1, s1, s2);
+            ADC_W(s3, s1, s2);
 
         IFX (X_ALL) {
             if (rex.w)
                 X64_ADC_D(s1, s2);
             else
                 X64_ADC_W(s1, s2);
         }
-
+        MV(s1, s3);
```

Comment:
I would duplicate the code to avoid the MV in the case there is no flags... but that's fine.